### DISCUSSION TITLE
Adding points to existing model

### DISCUSSION THREAD

**Discussion by ghilios** [mworion/MountWizzard4#141] [Q&A] [answered]
Is it possible to add points to an existing model? I have a model I built already with a gap in a certain part of the sky relevant for a target I'm shooting. I'd like to do a short model building run to cover just a few points in that spot of the sky, instead of building a full model from scratch. If I pick just a few spots and start a model run though, the resulting model only has those new points.

Thanks!

**Comment by mworion** [comment, marked as answer]
@ghilios 
Basically it is possible to do the additional points separately and afterward joining both (partly) models to a single one. This needs a mechanical unchanged setup! You need at least 3 points in each model, otherwise the mount computer rejects building a model.
For joining models, you could choose program model from file and select 2 model files.
Michel

**Reply by ghilios** [reply]
Thanks Michel, I'll try this on my next clear night!

**Reply by Vlandmist** [reply]
I had the same question. It can be useful to add a DSO track model to the existing model without reshooting all the sky. 
I will try to do this
